packages/generator/tests/utilities/measurements/non_streaming_data_measurement/__init__.py

```diff
@@ -134,6 +138,7 @@ def measure(
     enum_output = enum_input
     enum_array_output = enum_array_input
     protobuf_enum_output = protobuf_enum_input
+    double2d_array_output = array_pb2.Double2DArray()
```

Comment:
I'd like to see us return some sample data here so in the test we make sure all the data is coming through with its rows and columns.

Reply:
Why didn't we do this for DoubleXYData?

Reply:
Refer to this [comment](https://github.com/ni/measurement-plugin-python/pull/1020#discussion_r1871044195)